feathr_project/feathr/anchor.py

```diff
@@ -38,9 +38,7 @@ def validate_features(self):
                 if feature.key == [DUMMY_KEY]:
                     raise RuntimeError(f"For anchors of non-INPUT_CONTEXT source, key of feature {feature.name} "
                                        f"should be explicitly specified and not left blank.")
-        for feature in self.features:
-            assert feature.key_alias == self.features[0].key_alias
-
+        
```

Comment:
Why do we remove those asserts?

Reply:
I found situations like this in a feature anchor: 
![image](https://user-images.githubusercontent.com/23374871/165945122-bfe66ffb-f142-4363-a1e3-94ab42e3f141.png)

So I removed this assertion to make the code pass, but shall we modify this assertion instead?

Reply:
what's the keyAlias for all of them? Cant' see it with your screenshot.

Reply:
0：['DOLocationID']
1：[‘DOLocationID’,'DOLocationID']

Reply:
> what's the keyAlias for all of them? Cant' see it with your screenshot.

0：['DOLocationID']
1：[‘DOLocationID’,'DOLocationID']